backend/brass/src/runner.rs

```diff
@@ -0,0 +1,226 @@
+#![allow(unused_imports)]
+
+use crate::BrassEngine;
```

Comment:
I think for the sake of consistency the imports here should be converted to use a single convention (either have one `use` per line or group them).